common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -135,4 +135,14 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2017, 12, 13),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-acquisitions-epic-end-of-year-countdown",
+    "Test a countdown until the end of year",
+    owners = Seq(Owner.withGithub("joelochlann")),
+    safeState = Off,
+    sellByDate = new LocalDate(2018, 1, 10),
```

Comment:
Let's make sure we remember to turn this off on new years day...

Reply:
🍾 🎆 🤕

Reply:
Good point, I've set the expiry to 31st Dec so we don't need to worry about that